### PR TITLE
openshift sdn information

### PR DESCRIPTION
**Pull request by bfallonf**
Added a small amount of information about the SDN. 

@adellape Does this lineup with the stuff you were telling me today?

@rajatchopra I'll tag you in the trello card, but can I ask for a quick tech review? Thanks in advance!

### CODE REVIEW

File: architecture/networking.adoc [hunk 1]
```diff
@@ -12,3 +12,9 @@ toc::[]
 Kubernetes ensures that each pod is able to network with each other, and allocates each pod an IP address from an internal network. This ensures all containers within the pod behave as if they were on the same host. Giving each pod its own IP address means that pods can be treated like physical hosts or virtual machines in terms of port allocation, networking, naming, service discovery, load balancing, application configuration, and migration.
 
 Creating links between pods is unnecessary. However, it is not recommended that you have a pod talk to another directly by using the IP address. Instead, we recommend that you create a link:kubernetes_model.html#service[service], then interact with the service.
+
+== OpenShift SDN
+
+OpenShift uses software-defined networking (SDN) for running Docker clusters and ensures the best setup for your Kubernetes master so that it can talk to the node hosts of your OpenShift instance.
```
Comment (rajatchopra) [inline architecture/networking.adoc]: OpenShift deploys a software-defined-networking (SDN) approach for connecting Docker containers in an OpenShift cluster.

Note: SDN does not connect kube master with hode hosts. It connects containers within node hosts.

**Comment by bfallonf** [comment]
@rajatchopra Thanks for the suggestions. I've made changes that you can see in the preview. Is there anything else you feel should be included?

@adellape From your comments on the trello card it sounds like the capacity this needs to be documented is up in the air. I think you're right that at the least there should be a small paragraph of two, similar to what I've attempted, just context and architecture stuff. If there's anything you think is missing let me know.

**Comment by bfallonf** [comment]
@rajatchopra Have you had the time to take a look at this yet? I'm not sure what else to do here, and there's seems to be a debate as to how much the openshift_sdn will be included in future versions so I'm not sure where to go from here. Thanks.

File: architecture/additional_concepts/networking.adoc [hunk 2]
```diff
@@ -12,3 +12,9 @@ toc::[]
 Kubernetes ensures that each pod is able to network with each other, and allocates each pod an IP address from an internal network. This ensures all containers within the pod behave as if they were on the same host. Giving each pod its own IP address means that pods can be treated like physical hosts or virtual machines in terms of port allocation, networking, naming, service discovery, load balancing, application configuration, and migration.
 
 Creating links between pods is unnecessary. However, it is not recommended that you have a pod talk to another directly by using the IP address. Instead, we recommend that you create a link:kubernetes_model.html#service[service], then interact with the service.
+
+== OpenShift SDN
+
+OpenShift deploys a software-defined networking (SDN) approach for connecting Docker containers in an OpenShift cluster. The OpenShift SDN connects all containers within a node host.
```
Comment (rajatchopra) [inline architecture/additional_concepts/networking.adoc]: ~~The OpenShift SDN connects all containers within a node host.~~ The OpenShift SDN connects all containers across all node hosts.

**Comment by rajatchopra** [comment]
One comment above. Rest looks good.

**Comment by bfallonf** [comment]
@rajatchopra Thanks for that. I've made the change. 

@adellape You picked this up more than I did, is there anything else you'd like to see added here?

**Comment by adellape** [comment]
LGTM for now.

**Pull request merged**